src/components/MemorableSection.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <div class="template-container">
+    <h2 class="title">A memorable NAV address</h2>
+    <p class="subtext">Get the 34 character alphanumeric address from your wallet, and paste it into the field on the left above. Then create your own NavCoin Community OpenAlias address by typing your name on the right. Note: The system won't allow duplicate addresses.</p>
+    <div class="address-container">
+      <div class="heading-one">Full NavCoin Wallet Address</div>
+      <div class="heading-two">Address converted using OpenAlias</div>
+      <div class="row one">
+        <div class="heading-one">Full NavCoin Wallet Address</div>
+        <div class="address one">NYRqFrTMULcmK13rz1DAHqgf8PGSrfBduT </div>
+        <div class="arrow one">→</div>
+        <div class="heading-two">Address converted using OpenAlias</div>
+        <div class="name one"> miguel@nav.community</div>
+      </div>
+      <div class="row two">
+        <div class="address two">NTdsZrjjZbSUpF7VqkCeJ5xTm7Fu3b3AZn </div>
+        <div class="arrow two">→</div>
+        <div class="name two"> saori@nav.community</div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+
+export default {
+  name: 'MemorableSection',
+}
+
+</script>
+
+<style scoped>
+
+.template-container {
+  padding: 100px 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.title {
+  display: inline-block;
+  min-width: 140px;
+  font-size: 32px;
+}
+
+.subtext {
+  min-width: 200px;
+  max-width: 60vw;
+  text-align: left;
+  margin-bottom: 35px;
+}
+
+@media (min-width: 1290px ) {
+  .subtext {
+    max-width: 775px;
+  }
+}
+
+.arrow {
+  display: inline-block;
```

Comment:
Sugest changing this to 

```css
display: inline-flex;
justify-content: center;
align-items: center;
```

To make sure its centered.

Reply:
Done